com.github.AlizaMedicalImaging.AlizaMS.metainfo.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be contributed upstream.

Reply:
What should I do?

Reply:
Basically, submit it to https://github.com/AlizaMedicalImaging/AlizaMS/.